modules/polygon/src/polygon-utilities.d.ts

```diff
@@ -0,0 +1,80 @@
+
+type SegmentVisitor = (p1: number[], p2: number[], i1: number, i2: number) => void;
+
+type SegmentVisitorFlat = (p1x: number, p1y: number, p2x: number, p2y: number, i1: number, i2: number) => void;
+
+export const WINDING_CLOCKWISE: number;
```

Comment:
Nit; normally we do `export const WINDING = {CLOCKWISE: , ... };`

Use with `WINDING.CLOCK_WISE`.